### ISSUE TITLE
New Bug in Storj CLI , CLI-daemon, GUI as well

### ISSUE THREAD

**Issue by BernyMY** [Storj/bridge#351]
Lately i've encounter alot ping time out on my connection while i run CLI, GUI as well. 
I never had any problem before on my cli running in 15-30nodes.  But now, even if i run 5 nodes on cli, my connection start unstable and keep ping timeout .  And i tried closed all the cli and run solely on GUI with 3 nodes, the ping time out is already started.  
Things i've troubleshoot :
1) Downgrade Node.js from 6.9.5 back to 6.9.1 = still same , not solved
2) Downgrade PM2 from 2.4.0 back to 2.2.1 = still same , not solved
3) Fresh PC install everything from start = still same , not solved. 

So right now i cant even use GUI, CLI, nor Daemon as well to farm. It also affect when im browsing net.  Is there something wrong with the bridge ?
![ping time](https://cloud.githubusercontent.com/assets/24865357/23099212/77d7373a-f69c-11e6-9008-529a131e5f69.jpg)

**Comment by littleskunk** [comment]
Could be related to https://github.com/kadtools/kad-quasar/issues/27

**Comment by BlackDuck888** [comment]
I observed same. I think this problem is related to NAT performance of your router and the limited resources of ram and cpu to handle a large amount of changing connections. I am able to run near the same amount of nodes as you, going over 20 nodes the system/internet becomes unstable,laggy or the router loosing connetion or even crash. And this all within minutes after starting more than 20. I did a cross check with installation of a enterprice class router (sophos UTM) software within my esxi, i did this with the same line, and was able to run 1000+ nodes over 4h without any problems, but this router does not fits in private network (voip support etc), so i am back to 20 nodes. Last month i rented a large aws node for some hours, there were 2000+ nodes no problem, no lag etc. The kad problem can be the source of so many changing connections, but the at the and the router is not able to handle the number of connections even though the line limit is not reached. The next router thing is, they a not designed to handle a large amount on smal packages (pps - packages per second), in normal internet use a router gets larger packages with growing download rate.
